stacks/go/build/Dockerfile

```diff
@@ -7,7 +7,7 @@ LABEL io.buildpacks.stack.id=${stack_id}
 
 # Install Golang
 USER root
-RUN dnf install -y golang
+RUN dnf install -y golang --skip-broken
```

Comment:
Interested to know why --skip-broken was needed.  Is there often broken packages related to golang or something else in ubi8?

Reply:
@lkingland see discussion in #serverless-faas. Issue appears to be resolved now.